tests/unit/test_sync.py

```diff
@@ -0,0 +1,97 @@
+from datetime import datetime, timedelta, timezone
+from unittest import mock
+from unittest.mock import MagicMock
+
+import pytest
+
+from ctms.crud import schedule_acoustic_record
+from ctms.sync import CTMSToAcousticSync
+from tests.unit.test_crud import StatementWatcher
+
+CTMS_ACOUSTIC_CLIENT_ID = "CLIENT"
```

Comment:
These test-only constants are just used in ``CTMSToAcousticSync`` below, I think they can be used directly there, unless there is more planned in the future that may use them